src/EnergyPlus/RoomAirModelManager.hh

```diff
@@ -86,69 +86,13 @@ namespace RoomAirModelManager {
                         bool &Errorfound); // find zone number and node number based on the node name
 
     bool CheckEquipName(EnergyPlusData &state,
-                        std::string const &EquipType, // Equipment type
                         std::string const &EquipName, // Equipment Name
                         std::string &SupplyNodeName,  // Supply node name
                         std::string &ReturnNodeName,  // Return node name
                         int TotNumEquip,              // how many of this equipment type
-                        int TypeNum);                 // equipment type number
-
-} // namespace RoomAirModelManager
-
-struct RoomAirModelManagerData : BaseGlobalStruct
-{
-
-    bool GetUCSDDVDataFlag = true; // UCSD
-    bool GetAirModelData = true;   // Used to "get" all air model data
-    bool MyOneTimeFlag = true;
-    int CompNum = 0;
```

Comment:
All of these were local variables that migrated into state somehow and now went back where they belong.